frontend/express/public/core/device-and-type/templates/technologyHomeWidget.html

```diff
@@ -11,7 +11,7 @@
 							<div class="version-graph-block bu-p-3">
 								<div class="bu-columns version-graph-title">
 									<div class="bu-column">{{item2.name}}</div>
-									<div class="bu-column">{{item2.value}} | {{item2.percent}} % </div>
+									<div class="bu-column">{{item2.value}} Users | {{item2.percent}} % </div>
```

Comment:
Let's use a translation for "Users" rather than hardcoding it :slightly_smiling_face: